docs/Using-Fleet/fleetctl-CLI.md

```diff
@@ -189,6 +190,95 @@ spec:
   query: select * from processes
 ```
 
+## Using fleetctl with an API-only user
+
+Now that `fleetctl` and the Fleet server are configured, it can be helpful to create an API-only user to use when running automated workflows. An API-only user can be given a role based on the abilities it needs. The default access level is `Observer`. For more information on roles, see the [user permissions documentation](./Permissions.md#user-permissions).
+
+### Create an API-only user
+
+To create your new API-only user using `fleetctl user create` and picking a name, email, and password and setting the `api-only` flag to `true`:
+
+```
+fleetctl user create --name "API User" --email api@example.com --password temp!pass --api-only
+```
+
+### Reset the password
+
+When a new user is created, a password reset is needed before that user will be able to perform queries. Since an API-only user cannot log in to the Fleet UI, this needs to be done through the REST API. We'll be doing this through the terminal using `curl`. 
+
+First, log in to the new user account using `fleetctl login`. Once you're logged in successfully to the API-only user, set up a variable to hold the user's token:
+
+```
+token=$(fleetctl config get token | rev | cut -d ' ' -f 1 | rev)
+```
+
+Then use `curl` to send a required password reset request to the REST API through the terminal:
+
+```
+curl -d '{"new_password":"NewPassGoesHere"}' -H "Authorization: Bearer ${token}" -X POST https://fleet.corp.example.com/api/v1/fleet/perform_required_password_reset
+```
+
+If you see a response like this, the request was successful:
+
+```
+{
+  "user": {
+    "created_at": "2022-03-16T20:42:00Z",
+    "updated_at": "2022-03-16T20:42:00Z",
+    "id": 52,
+    "name": "API User",
+    "email": "api@example.com",
+    "force_password_reset": false,
+    "gravatar_url": "",
+    "sso_enabled": false,
+    "global_role": "observer",
+    "api_only": true,
+    "teams": []
+  }
+}
+```
+
+While the original token is no longer valid, it's never a bad idea to clear variables out once you're done with them:
+
+```
+unset token
+```
+
+### Use fleetctl as the new user
+
+Now that the password has been reset, you will need to log in again using the updated password with `fleetctl login`. You'll now be able to perform tasks using `fleetctl` as your new API-only user. 
+
+### Create a new context for your standard user (optional)
```

Comment:
@ksatter @DominusKelvin @eashaw @mike-j-thomas @Desmi-Dizney Just a couple of thoughts: (not a blocker to merging)

The text of this heading affects the direct URL that jumps to this heading on the page.  (It turns into something like `#create-a-new-context-for-your-standard-user-optional`  in the URL)

- We could consider striking `(optional)` from this heading.
- We could do something smart in the code that decides what these heading URL slugs are, maybe have it ignore `(optional)` when building slugs
- Orrrrrr, we could just bust out this whole section into a less prominent note. For example:

```md
You'll now be able to perform tasks using `fleetctl` as your new API-only user.

>  ### Switching users
>  If you would like to user your API user…
```

Also, it would be nice for the heading to speak more directly to the thing our dear reader is likely to be trying to do (switching users).  Since the URL and page title already has "fleetctl" in it, we can err on the side of a shorter heading here, and rely on that existing context.

Reply:
@ksatter After a quick reread, I'm not 100% sure about using a block quote (`> `) here per se-- up to you.  I'd base what you do off of how it looks on the page, which you've read more closely than me.

I think the heading text is more important.

Mostly just wanted to mention the consideration about heading URL slugs, aiming to write most headings in such a way that they resonate with what your reader is likely to be trying to do in _their words_ (rather than Fleet's terminology for something), and to draw your attention to the block quote trick (probably mostly for other situations).

Here's [an example of the block quote trick](https://sailsjs.com/documentation/concepts/internationalization) in action: 
<img width="870" alt="image" src="https://user-images.githubusercontent.com/618009/160219836-7e9e3504-0b7a-4bef-bbc6-1e4b7ae0b42d.png">

Reply:
(^Sorry for the wall of text)

Reply:
@mikermcneil Thanks for the info! 274th read through also lead to some small tone shifts.